src/content/CloudBoilerplateNet/Services/CachedDeliveryClient.cs

```diff
@@ -287,7 +359,13 @@ public async Task<TaxonomyGroup> GetTaxonomyAsync(string codename)
         {
             var identifierTokens = new List<string> { KenticoCloudCacheHelper.TAXONOMY_GROUP_SINGLE_IDENTIFIER, codename };
 
-            return await CacheManager.GetOrCreateAsync(identifierTokens, () => DeliveryClient.GetTaxonomyAsync(codename), GetTaxonomySingleDependency, ProjectOptions.CreateCacheEntriesInBackground);
+            // The "productionValidator" delegate does not have to do anything since a DeliveryException is thrown instead.
+            return await CacheManager.GetOrCreateAsync(
+                identifierTokens, 
+                () => DeliveryClient.GetTaxonomyAsync(codename), 
+                response => true, 
```

Comment:
I'd replace the `true` value with e.g. `response != null`. We know that currently the Delivery SDK won't return anything and will throw an exception instead. However, in the future that might change and this code will remain valid and understandable (even without the comment above).